### ISSUE TITLE
[Designer] Bug 40019163: [ExpenseReport]: Screen reader focus order is incorrect when navigate in scan mode on Total Expense Amount and its associated value.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7612]
### Problem Description

[Bug 40019163](https://microsoft.visualstudio.com/OS/_workitems/edit/40019163): [ExpenseReport]: Screen reader focus order is incorrect when navigate in scan mode on Total Expense Amount and its associated value.

User Experience:
Screen reader users will be impacted if UI structure is not conveyed properly via AT's

Test Environment:
Application: Adaptive Cards
OS version: 2004 (OS Build 21301.1000)
Application Version: 2101.28.1.0
 
Pre-Requisite: Launch the Adaptive Card application.

Repro steps:
1. Launch the Adaptive Card application and Enable Narrator.
2. Adaptive Card UWP Visualizer screen starts appearing. 
3. Navigate to ExpenseReport.Json button by using Tab key and activate it.
4. Expense approval adaptive card will be appeared on right sight of the pane.
5. Navigate to Expense approval card by using Tab key.
6. Switch narrator into scan mode.
7. Navigate to data.

Actual Result:
Screen reader focus order is incorrect when navigate in scan mode on Total Expense Amount and its associated value.
Screen reader focus is moving along the column items.

Expected Result:
Screen reader focus order should be correct when navigate in scan mode for Total Expense Amount and its associated value.
The scan mode focus should move along row wise.

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Card JSON

```JSON
Designer Bug
```

**Comment by vagpt** [comment]
Closing this issue as user is getting the correct information while navigating in scan mode.
